actions/watch.ts

```diff
@@ -220,32 +256,37 @@ async function placeOrder(order: any, api_url: string) {
     };
 
     // if the api_url doesn't contain localhost, post
-    if (!api_url.includes("localhost")) {
-      const { data } = await axios.post(`${api_url}/api/v1/orders`, postData, {
-        headers: {
-          "Content-Type": "application/json",
-          accept: "application/json",
-        },
-      });
-      console.log(`API response: ${data}`);
-    } else {
-      console.log(`API request: ${JSON.stringify(postData)}`);
+    console.log(`[placeOrder] API request with params:`, apiUrl, postData);
+    if (!apiUrl.includes("localhost")) {
+      const { status, data } = await axios.post(
+        `${apiUrl}/api/v1/orders`,
+        postData,
+        {
+          headers: {
+            "Content-Type": "application/json",
+            accept: "application/json",
+          },
+        }
+      );
+      console.log(`[placeOrder] API response`, { status, data });
     }
   } catch (error: any) {
+    const errorMessag = "[placeOrder] Error placing order in API";
     if (error.response) {
+      const { status, data } = error.response;
       // The request was made and the server responded with a status code
       // that falls out of the range of 2xx
-      console.log(JSON.stringify(error.response));
```

Comment:
this was creating a cyclic dependency error

Reply:
:no_bicycles: